js/localStorage.js

```diff
@@ -0,0 +1,14 @@
+const name = document.getElementById('name');
+const email = document.getElementById('email');
+const message = document.getElementById('message');
+const button = document.getElementById('submit');
+
+button.onclick = function () {
+  const user = name.value;
+  const mail = email.value;
+  const sms = message.value;
+
+  if (user && mail && sms) {
+    localStorage.setItem(user, mail, sms);
+  }
+};
```

Comment:
- [ ] [OPTIONAL] It would be better if you store the data as an object
> eg
```

if (user && mail && sms) {
    localStorage.setItem('user', JSON.stringify({
         fullname: user,
         email: mail,
         message: sms
    }));
  }
```